### PR TITLE
Correction select files for modules

### PR DESCRIPTION
**Pull request by clesaec**

### CODE REVIEW

File: src/main/java/com/github/yp44/SetPropertyMojo.java [hunk 1]
```diff
@@ -73,53 +72,58 @@ public void execute() throws MojoExecutionException {
     }
 
     private String getGitModifiedFiles() throws MojoExecutionException {
-        final File gitPath = project.getBasedir().toPath().resolve(".git").toFile();
+        final MavenProject root = this.findRoot();
 
+        final String moduleName = project.getBasedir().getName();
```
Comment (ypiel-talend) [inline src/main/java/com/github/yp44/SetPropertyMojo.java]: moduleName is never used.

File: src/main/java/com/github/yp44/SetPropertyMojo.java [hunk 2]
```diff
@@ -73,53 +72,58 @@ public void execute() throws MojoExecutionException {
...
+                    .flatMap(Set::stream)
+                    .filter(this::accept)
+                    .distinct()
+                    .map((String rel) -> this.toAbsolute(rel, root))
+                    .collect(Collectors.joining(","));
 
-            if (!sfiles.isEmpty()) {
-                sfiles = sfiles.substring(0, sfiles.length() - 1);
+            if (sfiles.isEmpty()) {
+                sfiles = "unexistingFile0001.xml";
```
Comment (ypiel-talend) [inline src/main/java/com/github/yp44/SetPropertyMojo.java]: Should use "emptyListValue" option value

**Review by ypiel-talend — CHANGES_REQUESTED** [review]
Some minor requested changes

File: src/main/java/com/github/yp44/SetPropertyMojo.java (hunk 2 above)
Comment (clesaec) [reply src/main/java/com/github/yp44/SetPropertyMojo.java]: done

File: src/main/java/com/github/yp44/SetPropertyMojo.java (hunk 1 above)
Comment (clesaec) [reply src/main/java/com/github/yp44/SetPropertyMojo.java]: removed

**Comment by clesaec** [comment]
changes done after test on connectors-ee

**Comment by clesaec** [comment]
Work well when modify pom.xml like this
add modifiedFiles properties under properties 
`<modifiedFiles></modifiedFiles>`

Add plugin 
```XML
            <plugin>
                <groupId>com.github.yp44</groupId>
                <artifactId>modifiedfiles-maven-plugin</artifactId>
                <version>0.1-SNAPSHOT</version>
                <executions>
                    <execution>
                        <id>apply</id>
                        <phase>process-classes</phase>
                        <goals>
                            <goal>set-property</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
```

Then add modifiedFiles variable on spotless maven plugin : 
```XML
                    <java>
                        <encoding>UTF-8</encoding>
                        <includes>
                            <include>${modifiedFiles}</include>
                        </includes>
                        <licenseHeader>
```

**Comment by ypiel-talend** [comment]
> Work well when modify pom.xml like this
> add modifiedFiles properties under properties
> `<modifiedFiles></modifiedFiles>`
> 
> Add plugin
> 
> ```
>             <plugin>
>                 <groupId>com.github.yp44</groupId>
>                 <artifactId>modifiedfiles-maven-plugin</artifactId>
>                 <version>0.1-SNAPSHOT</version>
>                 <executions>
>                     <execution>
>                         <id>apply</id>
>                         <phase>process-classes</phase>
>                         <goals>
>                             <goal>set-property</goal>
>                         </goals>
>                     </execution>
>                 </executions>
>             </plugin>
> ```
> 
> Then add modifiedFiles variable on spotless maven plugin :
> 
> ```
>                     <java>
>                         <encoding>UTF-8</encoding>
>                         <includes>
>                             <include>${modifiedFiles}</include>
>                         </includes>
>                         <licenseHeader>
> ```
Not the configuration explained in README for spotless.

**Comment by clesaec** [comment]
So, i modify code or README ?

**Review by ypiel-talend — APPROVED** [review]
LGTM

**Pull request merged**